acceptance-tests/support/steps/input-steps.ts

```diff
@@ -0,0 +1,40 @@
+import {When, Then} from "@cucumber/cucumber";
+import {enterTextIntoTextInput, checkErrorMessageDisplayedForField, clickSubmitButton} from "./shared-functions";
+
+const fields: Record<string, string> = {
+    name: "userName",
+    email: "emailAddress",
+    password: "password",
+    "security code": "securityCode",
+    "department name": "department",
+    "mobile phone number": "mobileNumber"
+};
+
+When("they submit the {} {string}", async function (fieldName, value) {
+    await enterTextIntoTextInput(this.page, value, fields[fieldName]);
+    await clickSubmitButton(this.page);
+});
+
+When("they enter the {} {string}", async function (fieldName, value) {
+    await enterTextIntoTextInput(this.page, value, fields[fieldName]);
+});
+
+When("the user submits the security code", async function () {
+    await enterTextIntoTextInput(this.page, "123456", "securityCode");
+    await clickSubmitButton(this.page);
+});
+
+When("they submit their mobile phone number", async function () {
+    await enterTextIntoTextInput(this.page, "0770 9000 124", "mobileNumber");
+    await clickSubmitButton(this.page);
+});
+
+Then("the error message {string} must be displayed for the Your name input field", async function (errorMessage) {
```

Comment:
```suggestion
Then("the error message {string} must be displayed for the {} field", async function (errorMessage) {
```
Use the same method to access the field name